src/daq/src/WaveformAnalysis.cc

```diff
@@ -17,29 +20,44 @@ WaveformAnalysis::WaveformAnalysis(std::string analyzer_name) {
   fThreshold = fDigit->GetD("voltage_threshold");
   fSlidingWindow = fDigit->GetD("sliding_window_width");
   fChargeThresh = fDigit->GetD("sliding_window_thresh");
+  fRunFit = fDigit->GetI("run_fitting");
+  fFitWindowLow = fDigit->GetD("fit_window_low");
+  fFitWindowHigh = fDigit->GetD("fit_window_high");
+  fFitShape = fDigit->GetD("lognormal_shape");
+  fFitScale = fDigit->GetD("lognormal_scale");
```

Comment:
Sorry I missed this earlier -- there should be a check. If `run_fitting=false` users should be allowed to use the analyzer without specifying these parameters.